sidecar/nginx/lua/amalgam8.lua

```diff
@@ -498,6 +501,8 @@ function Amalgam8:update_state(input)
    local a8_actions = {}
    local err
 
+   self.myip = self:os_cmd("ifconfig eth0 | grep 'inet addr:' | cut -d: -f2 | awk '{ print $1}'", false)
```

Comment:
i had moved it to this position for the possibility that the IP address might change. i can move it back though